ProblemsSolution/obj/ProblemsSolution.csproj.nuget.cache

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I am preatty sure this `obj` folder should have been excluded.

Reply:
Fixed I probably forgot to remove the cached files after fixing the .gitignore.